Kwc/Advanced/IntegratorTemplate/Embed/ContentSender.php

```diff
@@ -9,8 +9,8 @@ protected function _render($includeMaster, &$hasDynamicParts)
         $ret = preg_replace('#(href|src|action)=(["\'])(/[^/])#', '$1=$2' . $domain . '$3', $ret);
         $up = Kwf_Config::getValue('uniquePrefix');
         $up = $up ? $up.'-' : '';
-        $class = str_replace('kwfUp-', $up, Kwf_Component_Abstract::formatRootElementClass($this->_data->componentClass, '').'Master');
-        $ret = preg_replace('#<body class="([^"]+)"#', '<body class="\\1 '.$class.'" data-'.$up.'base-url="'.$domain.'" ', $ret);
+        $class = str_replace('kwfUp-', $up, Kwf_Component_Abstract::formatRootElementClass($this->_data->componentClass, ''));
+        $ret = preg_replace('#(<body.*>)#', '${1}<div class="' . $class . '" data-'.$up.'base-url="'.$domain.'" />', $ret);
```

Comment:
.* frisst alles weg, auch > (\n nicht, darum hat es glücklicherweise bei dir funktioniert)

entweder `.*?` (ungreedy) oder `[^>]*` dann hast du das problem nicht

Reply:
die Zeilen ganz weg, dafür in Template ausgeben